src/main/java/com/uber/cadence/internal/worker/ActivityWorker.java

```diff
@@ -142,7 +148,23 @@ public void resumePolling() {
     }
   }
 
-  private class TaskHandlerImpl implements PollTask.TaskHandler<PollForActivityTaskResponse> {
+  private class MeasurableActivityTask {
+    PollForActivityTaskResponse task;
+    Stopwatch sw;
+
+    MeasurableActivityTask(PollForActivityTaskResponse task, Stopwatch sw) {
+      Objects.requireNonNull(task);
+      Objects.requireNonNull(sw);
+      this.task = task;
```

Comment:
this.task = Objects.requireNotNull(task);

Reply:
Done.